uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-popup-details-tab.component.js

```diff
@@ -162,31 +153,117 @@ define([
     }());
 
     /**
-     * Calculate change in balance, it updates local balance variables.
-     *
-     * @return {Promise} empty promise if all required params are not set
-     *   otherwise promise from server
+     * Adjusts dates in UI.
+     * "To" date must not be earlier than "From" date in case of a Single day request.
+     * "To" date must be after "From" date in case of a Multiple day request.
      */
-    function calculateBalanceChange () {
-      vm._setDateAndTypes();
-      vm._toggleBalance();
+    function adjustDatesInUI () {
+      var toDate = moment(vm.uiOptions.toDate);
```

Comment:
@AkA84 please explain why. I don't really see a reason. We don't do

```js
vm.requestInstance // just because it is an instance
```

Reply:
because you want to know at a glance if a certain date object is a moment or a native one. The same reason it's custom to have variables beginning with `$` if they are jQuery objects

Reply:
Done.